service/matching/handler.go

```diff
@@ -29,12 +29,12 @@ import (
 	"sync"
 	"time"
 
+	"google.golang.org/protobuf/proto"
```

Comment:
I occasionally wiggled import order around to match the convention of:

1. external dependencies
2. internal dependencies (think temporalio/api)
3. same repo dependencies